integration-tests/known_networks.go

```diff
@@ -154,7 +154,7 @@ var (
 		ChainID:                   80001,
 		Simulated:                 false,
 		ChainlinkTransactionLimit: 5000,
-		Timeout:                   time.Minute,
+		Timeout:                   blockchain.JSONStrDuration{Duration: time.Minute},
```

Comment:
Doesn't need to happen now, especially as we're likely to change this again, but `blockchain` seems like a sub-optimal package for this.